src/server/index.js

```diff
@@ -2,14 +2,12 @@ import config from 'config';
 
 import * as steem from 'steem';
 
-delete process.env.BROWSER;
-
 const path = require('path');
 const ROOT = path.join(__dirname, '../..');
 
 // Tell `require` calls to look into `/app` also
 // it will avoid `../../../../../` require strings
-process.env.NODE_PATH = path.resolve(__dirname, '..');
+Object.assign(process.env, {NODE_PATH: path.resolve(__dirname, '..')});
```

Comment:
why the Object.assign rather than the direct assignment?

Reply:
Because the plugin replaces both process.env.NODE_PATH and process.env['NODE_PATH'], Object.assign is the only one way I found that is unaffected

Reply:
It would be worth adding a comment about that above this change. You could save someone time next year.

Reply:
@valzav ?

Reply:
@relativityboy good point, I've added a comment

Reply:
Thanks.